haystack/nodes/answer_generator/openai.py

```diff
@@ -68,6 +71,9 @@ def __init__(
     ):
         """
         :param api_key: Your API key from OpenAI. It is required for this node to work.
```

Comment:
Let's update the `api_key` docstring to mention that the OpenAI API or Azure API key is needed.